go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/opencontainers/runtime-spec
```

Comment:
This has a potential wide impact than updating go versions in CI; do we still need to discuss;

https://github.com/opencontainers/runtime-spec/issues/1069

(or was there a concensus on that yet w.r.t. "SemVer from a Go perspective vs Spec perspective")?

Reply:
image-spec and distribution-spec already have `go.mod`:
- https://github.com/opencontainers/image-spec/blob/main/go.mod
- https://github.com/opencontainers/distribution-spec/blob/main/specs-go/go.mod

And these struct definitions are unlikely to have breaking changes (until the spec itself has breaking changes)

Reply:
@thaJeztah WDYT?

Reply:
@austinvazquez @thaJeztah 
Do we need to split `go.mod` to another PR?

Reply:
I would consider moving it separate yes, as it's not directly related to updating / updating CI to test against a newer go version

Reply:
Okay, thanks for the feedback. Let me split and push.